### ISSUE TITLE
IE11 : ellipsis still not showing correctly using ibm-plex-sans font. 

### ISSUE THREAD

**Issue by dsuwirya** [carbon-design-system/carbon#1079]
If you visit the www.carbondesignsystem.com website today, using IE11, and tweak it so it displays ellipsis on any part of the page ( which is using ibm-plex-sans font ), you'll notice that the ellipsis is still not showing correctly ( see attached image ).

Ref: 
- https://github.com/IBM/carbon-components/issues/665
- https://github.com/IBM/plex/issues/110

![image](https://user-images.githubusercontent.com/5710527/45367954-fcc90b00-b5a7-11e8-9171-8ad1ebe12247.png)

**Comment by asudoh** [comment]
Issue moved to [IBM/design-system-website #352](https://github.com/IBM/design-system-website/issues/352) via [**ZenHub**](https://www.zenhub.com/)

**Comment by dsuwirya** [comment]
I just tried again today, and it's still the same issue. I tried it on the dropdown component itself now. Please see attached image.

![image](https://user-images.githubusercontent.com/5710527/45497072-7abe1b00-b73c-11e8-8ca8-4fa4424f5310.png)


![image](https://user-images.githubusercontent.com/5710527/45497234-e607ed00-b73c-11e8-81d1-384f4fd647b2.png)


![image](https://user-images.githubusercontent.com/5710527/45497271-fddf7100-b73c-11e8-947f-fe6f6796f69e.png)

**Comment by tw15egan** [comment]
@dsuwirya the fix has been made, but not sure the fix has been deployed to the website yet.

**Comment by asudoh** [comment]
@tw15egan is right - Next deployment will solve the issue. Thanks!

**Comment by dsuwirya** [comment]
@asudoh Can you please ping me when the next deployment happens so I can try it out again ? It's not that I'm interested with the website fix, but I'm more interested to know if #665 is really resolved or not, as it doesn't seem to be the case on our side. And if it does, I then need to dig further on our side to find out what's the difference. Thanks!

**Comment by asudoh** [comment]
@dsuwirya In such case, the best way for you to verify if the problem is fixed is:

```sh
> git clone git@github.com:IBM/carbon-components.git
> cd carbon-components
> yarn install
> gulp serve
```

And hit `localhost:3000` in IE11.

Hope it helps!
